tox.ini

```diff
@@ -36,45 +58,38 @@ passenv =
     PASS_INVALID_VALUES
     VALIDATE_ON_ASSIGNMENT
     TEST_BIGDATA
-
 usedevelop = true
-
 commands =
-    !cov: pytest -n auto {posargs}
-    cov: pytest -n auto --cov-report=xml --cov=. --cov-config=setup.cfg {posargs}
-
+    pip freeze
+    pytest -n auto \
+    cov: --cov=. --cov-config=setup.cfg --cov-report=term-missing --cov-report=xml \
+    warnings: -W error \
+    regtests: --bigdata --slow --basetemp={homedir}/test_outputs \
+    {posargs}
```

Comment:
Is the `\` in each of the 4 lines all pointing to `{posargs}`?  This is a confusing syntax.  Wouldn't be clearer to have:

```
pytest -n auto {posargs}
cov: --cov=. --cov-config=setup.cfg --cov-report=xml {posargs}
...
```

etc?

Reply:
These are so that several different args might be added or not added; the line might be any one of 
```
pytest -n auto {posargs}
pytest -n auto --cov=. --cov-config=setup.cfg --cov-report=term-missing --cov-report=xml {posargs}
pytest -n auto --cov=. --cov-config=setup.cfg --cov-report=term-missing --cov-report=xml -W error {posargs}
pytest -n auto -W error {posargs}
pytest -n auto -W error --bigdata --slow --basetemp={homedir}/test_outputs {posargs}
```
etc.

Reply:
Ah, that's clever!  Carry on.  🚀